pkgs/by-name/xa/xan/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "xan";
+  version = "0.42.0";
+
+  src = fetchFromGitHub {
+    owner = "medialab";
+    repo = "xan";
+    tag = version;
+    hash = "sha256-8uNfiPJjZWqtbrBU/LRYM7PI5/4JBqZGe1dtCWArU8M=";
+  };
+
+  cargoHash = "sha256-4OVDiXa8iutoPr816yjMFYa+bC5vZ+iHHrij7uwYUZ0=";
+  enableParallelBuilding = true;
+
+  # FIXME: tests fail and I do not have the time to investigate. Temporarily disable
+  # tests so that we can manually run and test the package for packaging purposes.
+  doCheck = false;
+
+  meta = {
+    description = "The CSV magician";
```

`meta.description` must (see [docs](https://github.com/NixOS/nixpkgs/blob/479d8786151352de75ea8198ec40624ee18f1f00/pkgs/README.md#meta-attributes)):
- Not start with the definite article.
- Provide factual information.

I don't think it's really clear what a "CSV magician" is. Instead, something like this might be better:

```suggestion
    description = "Command line tool to process CSV files directly from the shell";
```

---

Sounds about right, I was hoping to revisit the description later on.